src/app.md

```diff
@@ -271,6 +272,40 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot]: #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterates over filesystem entries efficiently skipping hidden files and directories with help of [WalkDir::filter_entry]. 
```

Comment:
> mm for me it works

It does not for github markdown renderer and mdbook's pulldown-cmark (which is used for final rendering on releases). How do you verify?